File: crates/harn-hostlib/tests/fixtures/ast/php/source.php
<?php

class Greeter {
    private string $name;

    public function __construct(string $name) {
        $this->name = $name;
    }

    public function greet(): string {
        return "hello " . $this->name;
    }
}

interface Speaker {
    public function speak(): string;
}

function shout(string $s): string {
    return strtoupper($s);
}

File: crates/harn-hostlib/tests/fixtures/ast/go/source.go
package fixture

type Greeter struct {
	Name string
}

type Speaker interface {
	Speak() string
}

func (g Greeter) Greet() string {
	return "hello " + g.Name
}

func Shout(s string) string {
	return s
}

func TestSomething() {}
